RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -0,0 +1,331 @@
+import numpy
+
+from RUFAS.biophysical.animal.digestive_system.methane_mitigation_calculator import MethaneMitigationCalculator
+from RUFAS.general_constants import GeneralConstants
+
+
+class EntericMethaneCalculator:
+    @staticmethod
+    def calf_methane(methane_model: str | None, body_weight: float) -> float:
+        """
+        Calculates the amount of methane emission for calf.
+
+        Parameters
+        ----------
+        methane_model: str
```

Comment:
```suggestion
        methane_model: str | None
```
I think it would be good to note in this docstring (probably for this parameter) that if the methane_model is none then the methane emissions are set as zero